cli/cli.go

```diff
@@ -32,6 +33,11 @@ func Run() {
 			Value: "info",
 			Usage: fmt.Sprintf("Log level (options: debug, info, warn, error, fatal, panic)"),
 		},
+
+		cli.BoolFlag{
+			Name:  "experimental",
```

Comment:
Yes that was my point,
docker engine for example has one regular build and one experimental build. not one build per experimental feature.

Going forward, ppl who wants to test the bleeding edge stuff just enable "-experimental" and that's it, they don't have to remove/add flags for every release.